.github/test-spec.yml

```diff
@@ -591,9 +591,7 @@
   - "samples/event_manager_proxy/**/*"
   - "snippets/emulated-*/**/*"
   - "tests/subsys/event_manager_proxy/**/*"
-  - "tests/benchmarks/current_consumption/**/*"
-  - "tests/benchmarks/multicore/**/*"
-  - "tests/benchmarks/peripheral_load/**/*"
```

Comment:
Was this stress test intentionally removed? It has nothing to do with idle and power states.

Reply:
Yes, as  it is not checked at integration downstream, thus should not trigger downstream.